charts/akeyless-secure-remote-access/templates/akeyless-zero-trust-portal/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if eq .Values.ztpConfig.enabled true }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $.Release.Name }}-cm-ztp
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ $.Values.ztpConfig.containerName }}
+    chart: {{ $.Chart.Name }}
+data:
+  env-config.js: |
+    window._env_ = {
```

Comment:
will it always be "window._env_" or other configuration can be applied?

Reply:
only this one